### DISCUSSION TITLE
Reporting a Vulnerability

### DISCUSSION THREAD

**Discussion by AmirMSafari** [axios/axios#6184] [Q&A]
Hello everyone! I've come across an unusual issue that seems to have potential security implications. I'd like to report it using the method outlined in the `SECURITY.md` file at `https://huntr.dev/repos/axios/axios`. However, it appears that the platform is not currently accepting reports. Can anyone suggest an alternative channel where we could discuss and address this matter? Your assistance is greatly appreciated

**Comment by jasonsaayman** [comment]
Please use the security tab. Also, check all the docs in the repo.
